internal/cmd/ozctl/cmd/create_exec_access_request.go

```diff
@@ -108,7 +108,7 @@ func init() {
 	createExecAccessRequestCmd.Flags().
 		StringVarP(&waitTime, "wait", "w", "1m", "Duration to wait for the access request to be fully ready. Valid time units are: ns, us, ms, s, m, h.")
 	createExecAccessRequestCmd.Flags().
-		StringVarP(&requestNamePrefix, "request-name", "N", usernameEnv, "Prefix name to use when creating the `ExecAccessRequest` objects.")
+		StringVarP(&requestNamePrefix, "request-name", "N", fmt.Sprintf("oz-%s", usernameEnv), "Prefix name to use when creating the `ExecAccessRequest` objects.")
```

Comment:
1. Labels should be used to isolate/filter for pods... not names.
2. This should be a configurable property that is optional. The question is, shoudl it be a controller setting or an AccessTemplate setting?